src/steps/AddAnother.js

```diff
@@ -138,6 +142,18 @@ class AddAnother extends Question {
     res.redirect(this.path);
   }
 
+  buildValidationErrors(fields) {
```

Comment:
Just a style suggestion but you can access the fields from inside this function so you could write it as a getter:

```
get validationErrors() {
  const fieldList = Object.keys(this.fields);
  ...
}
```

Suggest avoiding using `this.locals` very much, it's an expensive function that we can't avoid on rendering a template but you could avoid here.